recipes/seareport-data/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "seareport-data" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/seareport_data-{{ version }}.tar.gz
+  sha256: 74e632d09f211318658c5a9e3f7605b5e5d60a5c50d16286ab68382eb9afe57b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10,<4.0
```

Comment:
This looks generally good, but could you satisfy the linter by following the docs on handling `noarch: python` recipes:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

You can set a jinja variable for `python_min` specifying 3.10

Reply:
Done